egs/librispeech/asr/simple_v1/generate_bpe_tokens.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python3
+
+# Copyright (c)  2021  Xiaomi Corporation (authors: Fangjun Kuang)
+'''
+Example usage of this script:
+
+python3 ./generate_bpe_tokens.py \
+  --model-file ./data/lang_bpe/bpe_unigram_500.model > data/lang_bpe/bpe_unigram_500.tokens
```

Comment:
Is it necessary to write the model out to disk and get the tokens separately?
I'd imagine it would be very fast to train the BPE model (isn't it some very simple LM?) and get the tokens inside the same script..  this just feels a bit complicated.   But it's OK for now.

Reply:
> Is it necessary to write the model out to disk and get the tokens separately?

No, it's not necessary. We can generate all the needed files `tokens.txt`, `words.txt`, `lexicon.txt` in one
file. I just feel splitting them is easier to understand.

Yes, it is quite fast to train a BPE model. It's less than 1 minute for the LibriSpeech dataset.
The model is only several hundred KB and does not occupy too much disk space.

Reply:
> Is it necessary to write the model out to disk 

According to the documentation and examples given in
https://github.com/google/sentencepiece/blob/master/python/README.md ,
it looks like the model has to be written to disk after training. There is no option to retain a memory representation
of the model without writing it to disk.